lib/dry/rails/features/safe_params.rb

```diff
@@ -33,7 +33,7 @@ def schema(*actions, &block)
               schemas[name] = schema
             end
 
-            before_action(:set_safe_params, only: actions)
+            before_action(:set_safe_params)
```

Comment:
@gotar I *think* we want to move this to the `self.included` hook + tweak `set_safe_params` so that it checks if there is a schema defined for the current action.

Reply:
Hmm ok will try to tweak it later. We will see

Reply:
Maybe something like that, check last commit, but works without all of this tweaks too.